README.md

```diff
@@ -192,6 +192,39 @@ Verify that the `asciidoctor-web-pdf` command is available on your `PATH` by run
 $ yarn global add @asciidoctor/core asciidoctor-pdf
 ```
 
+### Using Docker
+
+Currently, there is no public image yet pushed in a public docker registry.
+
+Therefore, you can build your own custom docker image.
+
+The base image is from the official [puppeteer](https://github.com/puppeteer/puppeteer/pkgs/container/puppeteer), which contains puppeteer and google-chrome for pdf generation.
+
+To build the docker image. The target image name in this example is `adoc2web`.
+
+```bash
+cd docker
+docker build . -t adoc2web
+```
+
```

Comment:
```suggestion

The target image name in this example is `adoc2web`.
```

I think it would be better to use the name `asciidoctor-web-pdf-latest` or something

Reply:
Yes it would be more semantic.